From 70415a8f13300050d178405fffc119a743877ffc Mon Sep 17 00:00:00 2001
From: Ecco park <eccopark@broadcom.com>
Date: Mon, 4 Aug 2014 15:54:23 -0700
Subject: [PATCH 41/77] net : wireless: bcmdhd remove skb_orphan due to side
 effect

1) [CERT] WiFi :11ac test case 5.2.32.1 and 5.2.33.1 failures
2) The code causes the regression of throughput

Bug: 16716038
Signed-off-by: Ecco park <eccopark@broadcom.com>
---
 drivers/net/wireless/bcmdhd/dhd_pcie.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/net/wireless/bcmdhd/dhd_pcie.c b/drivers/net/wireless/bcmdhd/dhd_pcie.c
index b04a46e6d79b..69cac784b89e 100644
--- a/drivers/net/wireless/bcmdhd/dhd_pcie.c
+++ b/drivers/net/wireless/bcmdhd/dhd_pcie.c
@@ -1527,8 +1527,6 @@ dhd_bus_schedule_queue(struct dhd_bus  *bus, uint16 flow_id, bool txs)
 		DHD_QUEUE_LOCK(queue->lock, flags);
 
 		while ((txp = dhd_flow_queue_dequeue(bus->dhd, queue)) != NULL) {
-			PKTORPHAN(txp);
-
 #ifdef DHDTCPACK_SUPPRESS
 		dhd_tcpack_check_xmit(bus->dhd, txp);
 #endif /* DHDTCPACK_SUPPRESS */
-- 
2.0.3

